charts/umbrella-charts/ondat/Chart.yaml

```diff
@@ -17,7 +17,7 @@ dependencies:
   repository: https://ondat.github.io/charts
   version: 0.7.3
 - name: etcd-cluster-operator
-  version: 0.1.4
```

Comment:
Can you please remove this change, we can't release both charts simultaneously so it's best to just touch one at once